testing/MLDB-951-run-on-creation.py

```diff
@@ -99,18 +99,21 @@ def load_test_dataset():
     assert int(row['rowName']) + 1 == row['columns'][0][1], 'the transform was not applied correctly'
 
 # checking return code for invalid runs
-response =  mldb.perform('PUT', "/v1/procedures/transform_procedure", [], 
+res =  mldb.perform('PUT', "/v1/procedures/transform_procedure", [], 
                          {
                              "type": "transform",
                              "params": {
                                  "inputData": "select x + 1 as x from dataset1",
-                                 "outputDataset": { "id": "dataset2", "type":"beh" }, # try to create a non-mutable
+                                 "outputDataset": { "id": "dataset2", "type":"beh" }, # try to create a type that is not available
```

Comment:
I will change that to avoid the confusion.  After all, the only thing we want is to cause an error.